.python-version

```diff
@@ -0,0 +1 @@
+3.9.16
```

Comment:
nit: we as might as well use 3.12 -- this would also require changing the `run-unit-tests.yml` file